### ISSUE TITLE
v5.0.4 Crashlytics could not find the manifest. Not found at\mobile\build\intermediates\merged_manifests\debug\universal\AndroidManifest.xml

### ISSUE THREAD

**Issue by shyandsy** [shadowsocks/shadowsocks-android#2439]
**Describe the bug**
when I try to run the mobile app, theres is an error information shows below:

```
v5.0.4 Crashlytics could not find the manifest. Not found at\mobile\build\intermediates\merged_manifests\debug\universal\AndroidManifest.xml
```

**Comment by shyandsy** [comment]
@Mygod what is mean by invalid???

**Comment by Mygod** [comment]
This is an issue of Crashlytics. To get around it, build debug apks first.

**Comment by Mygod** [comment]
Fixed via 8e2c581.
